### ISSUE TITLE
Offline version no working properly

### ISSUE THREAD

**Issue by jegarfa** [tableaunoir/tableaunoir#251]
Hi, I tried to run the offline version of tableaunoir but its window did not load all the whiteboard elements properly (see the screenshot). I am unsure if this relates to the electron version or something else. 

I used the `$ electron mainElectron.js` on a LinuxMint 21.1 machine with updated npm and electron. Although I did not receive error messages in the terminal. Does anyone have a clue on what's happening and how to solve it?
Thanks!

![Screenshot from 2024-02-13 11-28-39](https://github.com/tableaunoir/tableaunoir/assets/10395230/658101ac-b25d-4565-bfef-37398140a205)

**Comment by francoisschwarzentruber** [comment]
Hello,

I will try to solve your issue. It works on my computer:
![Screenshot from 2024-02-13 18-45-02](https://github.com/tableaunoir/tableaunoir/assets/43071857/547326ae-ccdb-44e0-b919-20272f64cf66)

I will think about it and try to see why it is not working.

**Comment by francoisschwarzentruber** [comment]
I have two questions:
- did you compile the project with `npm run build`?
- what happen if you run `dist/index.html` in your browser?

**Comment by jegarfa** [comment]
Hi, I just tried compiling the project with `npm run build` and everything is working now.
I missed this step, in fact, I just noticed that I did not have the folder `dist`. 

Now the `dist/index.html` reproduce the interface with all the elements, thanks!

![Screenshot from 2024-02-13 20-35-53](https://github.com/tableaunoir/tableaunoir/assets/10395230/69ed376c-9097-476b-9f8a-a0d68ade44c1)

Thanks a lot!
